indexer/docs/api/withdrawals.md

```diff
@@ -0,0 +1,177 @@
+## User Withdrawals
```

Comment:
Withdraw endpoint.   

I considered 2 alternatives

1. Deposits and withdrawals more generalized endpoint with address being a filter 
2. Deposits and withdrawals being combined into `transactions` endpoint

This is open to changing if base gives feedback needing to change this.  Otherwise this api design is a very specific 1:1 mapping to how the UI works that can easily be generalized more in future if necessary.

Would very quickly consider making this graphql if it needed to be generalized